jmh-benchmarks/src/main/java/org/apache/kafka/jmh/assignor/ServerSideAssignorBenchmark.java

```diff
@@ -119,7 +123,13 @@ public enum AssignmentType {
 
     private SubscribedTopicDescriber subscribedTopicDescriber;
 
-    private final List<Uuid> allTopicIds = new ArrayList<>();
+    private final List<String> allTopicNames = new ArrayList<>();
+
+    private ImmutableMap<Uuid, TopicImage> topicsById = ImmutableMap.empty();
+
+    private ImmutableMap<String, TopicImage> topicsByName = ImmutableMap.empty();
+
+    private TopicsImage topicsImage = TopicsImage.EMPTY;
```

Comment:
I can't seem to use MetadataImageBuilder in jmh module as it is in the test directory. Do you know whether we can go around that?

Reply:
Hum... I guess that we cannot. In this case, it may be better to build the image the right way. We usually build it like we do in the MetadataImageBuilder.